docs/t-sql/statements/alter-index-transact-sql.md

```diff
@@ -252,11 +252,14 @@ PARTITION
  REORGANIZE a **rowstore** index  
  For rowstore indexes, REORGANIZE specifies to reorganize the index leaf level. The REORGANIZE operation is:  
   
--   Always performed online. This means long-term blocking table locks are not held and queries or updates to the underlying table can continue during the ALTER INDEX REORGANIZE transaction.  
+-   Always performed online. This means long-term blocking table locks are not held and queries or updates to the underlying table can continue during the ALTER INDEX REORGANIZE transaction.   
 -   Not allowed for a disabled index  
 -   Not allowed when ALLOW_PAGE_LOCKS is set to OFF  
 -   Not rolled back when it is performed within a transaction and the transaction is rolled back.  
 
+> [!NOTE]
+> If using explicit transactions instead of the default implicit transaction mode (e.g., ALTER INDEX within BEGIN TRAN ... COMMIT/ROLLBACK), the locking behavior of REORGANIZE becomes more restrictive. For more information on implicit transactions, see [SET IMPLICIT_TRANSACTIONS &#40;Transact-SQL&#41;](../../t-sql/statements/set-implicit-transactions-transact-sql.md)
```

Comment:
The way it reads now almost seems like implicit is inside a BGEIN/END TRAN. Suggest the following:

When ALTER INDEX REORGANIZE uses explicit transactions (e.g., ALTER INDEX inside a BEGIN TRAN ... COMMIT/ROLLBACK) instead of the default implicit transaction mode, the locking behavior of REORGANIZE becomes more restrictive. For more information on implicit transactions, see [SET IMPLICIT_TRANSACTIONS &#40;Transact-SQL&#41;](../../t-sql/statements/set-implicit-transactions-transact-sql.md).

Reply:
Do you guess that we can add following bold part to clarify how more restrictive it can be?

he locking behavior of REORGANIZE becomes more restrictive, **causing long-term blocking table locks**

Reply:
"long" is subjective - it's as long as the reorg takes. It may be short or not. I'd be ok with ", potentially causing blocking."

Reply:
Yes. I agree the word "potentially" is the right one for this. Thanks